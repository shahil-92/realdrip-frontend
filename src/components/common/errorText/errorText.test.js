import React from 'react';
import ErrorText from '.';

describe('<ErrorText />', () => {
  describe('Rendering', () => {
    it('should render nothing when there is no text prop', () => {
      const wrapper = shallow(<ErrorText />);
      expect(wrapper).toMatchSnapshot();
    });
    it('should render correctly when there is a text prop', () => {
      const wrapper = shallow(<ErrorText text="I am an error" />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});

