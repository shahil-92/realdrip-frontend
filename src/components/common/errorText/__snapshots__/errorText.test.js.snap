// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorText /> Rendering should render correctly when there is a text prop 1`] = `
<span
  className="error-text"
>
  I am an error
</span>
`;

exports[`<ErrorText /> Rendering should render nothing when there is no text prop 1`] = `""`;
